test/middleware/csrf.test.js

```diff
@@ -97,18 +99,11 @@ describe('retrieve param test', function () {
     assertUnauthorisedRequest(next, resp, status, render)
   })
 
-  it('should be successful on post if valid post and append token to used tokens', function () {
+  it('should be successful on post if valid post', function () {
     const resp = _.cloneDeep(response)
     csrfCheck(validPost, resp, next)
     csrfTokenGeneration(validGetRequest, resp, next)
     assertValidRequest(next, resp, status, render)
-    assert.strictEqual(validPost.frontend_state.ch_foo.csrfTokens[0], validPost.body.csrfToken)
-  })
-
-  it('should be unsuccessful on post if token is already used', function () {
-    const resp = _.cloneDeep(response)
-    csrfCheck(validPost, resp, next)
-    assertErrorRequest(next, resp, status, render)
   })
```

Comment:
I'm nervous about the removal of the csrf tests when that logic's changed (written before getting to the end - maybe they've been moved into a different file)

Reply:
this test has been removed because we no longer store used csrf tokens.